### ISSUE TITLE
Rename "Lobby" to "Boards"

### ISSUE THREAD

**Issue by TheFleetCommander** [hellopaint-io/hellopaint#29]
Current: In order to access the main "Boards" dashboard, I need to click on a button labeled "Lobby". This is confusing because it's unclear what Lobby means in this context since all of the components refer specifically to "Boards". As a user, I'd expect a Lobby to be more like a global chat-room or a single default board.

![image](https://user-images.githubusercontent.com/131052789/233213719-1e6370ba-8506-412b-80c5-44a88e4efa62.png)
![image](https://user-images.githubusercontent.com/131052789/233214144-229d2f64-693f-42bd-94c0-0ff552e97999.png)

Expected: Rename from "Lobby" to "Boards". This resolves the above inconsistency while also making HP's UI more consistent with Malmal's navigation.

**Comment by lucasmerlin** [comment]
I agree, I think we initially called it lobby because there was a lobby on iScribble, but Boards does make more sense for HP

**Comment by Krowphay** [comment]
I agree as well. The lobby on iScribble wasn't only to show boards, but also to show a list of all online users. If I recall correctly you could chat there directly, but people wouldn't see it unless they were also in the lobby. I like the idea of making an infinite canvas "lobby" board similar to Malmal's Newcomers 2.0 that's accessible from such a button but even that I think might be most appreciated if it's just always at the top of the boards tab instead, like a featured board?
